tests/inputFields.spec.ts

```diff
@@ -59,16 +59,17 @@ test.describe('Pet types', () => {
     })
 
     test('Pet type name validation', async ({page}) => {
-        const petTypesTableName = page.getByRole('heading', {name: "Pet Types"})
+        const petTypesTableName = page.locator('app-pettype-list h2')
         await expect(petTypesTableName).toHaveText('Pet Types')  //2. assertion of Pet Types text displayed
 
-        await page.getByRole('button', {name: "Edit"}).nth(2).click() //3. click on edit button for lizard
-        const editPetTypeName = page.locator('#name') //4. clear input field
-        await editPetTypeName.click()
-        await editPetTypeName.clear()
+        const petTypeTableEditButton = page.getByRole('button', {name: "Edit"})
+        await petTypeTableEditButton.nth(2).click() //3. click on edit button for lizard
+        const editPetTypeNameInputField = page.locator('#name') //4. clear input field
```

Comment:
can be replaced with a single line: 
`await page.getByRole('button', {name: "Edit"}).nth(2).click()`